_component_design/text-input.md

```diff
@@ -11,7 +11,7 @@ status: stable
 last-modified: 2018-10-26
 helix-ui-css: true
 helix-ui-javascript: na
-pagelink: https://rackerlabs.github.io/helix-ui/components/text-inputs/
+pagelink: https://rackerlabs.github.io/helix-ui/components/text-input/
```

Comment:
We split `components/text-inputs` (plural; input + textarea) into `components/text-input` (singular; input only) and `components/textarea` (singular; textarea only). Is this going to be a problem?

Reply:
Hm. It just means people will get linked to the singular input when coming from the status page. I think we should be ok until we make it to thunderdome.